# config/vlp16_lexus_centroid.param.yaml
---

voxel_grid_cloud_node:
  ros__parameters:
    input_topic: "lidar_front/points_transformed"
    downsample_topic: "points_downsampled"
    is_approximate: false
    config:
      capacity: 55000
      min_point:
        x: -130.0
        y: -130.0
        z: -3.0
      max_point:
        x: 130.0
        y: 130.0
        z: 3.0
      voxel_size:
        x: 2.0
        y: 2.0
        z: 2.0
